plugins/woocommerce/src/Internal/ComingSoon/ComingSoonRequestHandler.php

```diff
@@ -52,13 +52,20 @@ public function handle_template_include( $template ) {
 
 		$coming_soon_template = get_query_template( 'coming-soon' );
 
-		if ( ! wc_current_theme_is_fse_theme() && $this->coming_soon_helper->is_store_coming_soon() ) {
+		$is_fse_theme = wc_current_theme_is_fse_theme();
+		$is_store_coming_soon = $this->coming_soon_helper->is_store_coming_soon();
+
+		if ( ! $is_fse_theme && $is_store_coming_soon ) {
 			get_header();
 		}
 
+		add_action('wp_head', function() {
+			echo "<meta name='woo-coming-soon-page' content='yes'>";
+		});
+
 		include $coming_soon_template;
 
-		if ( ! wc_current_theme_is_fse_theme() && $this->coming_soon_helper->is_store_coming_soon() ) {
+		if ( ! $is_fse_theme && $is_store_coming_soon ) {
 			get_footer();
 		}
 
```

Comment:
There's another opportunity to use `$is_fse_theme` in the line below this

Reply:
@psealock Good catch! Fixed in [Reuse $is_fse_theme](https://github.com/woocommerce/woocommerce/pull/48586/commits/8ff1ee394af86054f9e8d46c9996fd90d9773d4f)